content/tutorials/webrtc/datachannels/en/index.html

```diff
@@ -219,7 +220,7 @@ <h2 id="build-a-file-sharing-application">Build a file sharing application</h2>
 <ul>
   <li><strong>File size:</strong> if file size is reasonably small and can be stored and loaded as one Blob, you can load into memory using the File API and then send the file over a reliable channel as is (though bear in mind that browsers impose limits on maximum transfer size). As file size get larger, things get trickier. A chunking mechanism is required: file chunks are loaded and sent to another peer, accompanied with chunkId metadata so the peer can recognize them. Note that in this case you will also need to save the chunks first to offline storage (for example, using the FileSystem API) and only when you have the file in its entirety save it to the user's disk.</li>
   <li><strong>Speed:</strong> it is debatable whether unreliable (UDP-like) or reliable (TCP-like) transport is better for the job of file transfer. If the application is a simple one-to-one file transfer, going with unreliable data channels will take some design effort for the ACK/retransmit protocol. You will need to implement this yourself and &mdash; even if you're good &mdash; it may not be much faster than using reliable transport. Reliable and unordered should provide the best of both worlds, but results may vary when considering multi-party (mesh) file transfer.</li>
```

Comment:
> Reliable and unordered should provide the best of both worlds, but results may vary when considering multi-party (mesh) file transfer.

This statement is a bit puzzling to me. Can you elaborate why that should be the case?

Reply:
remove?

Reply:
Possibly. But I would like to understand the background first. :)

Reply:
assume someone explained it wrong to @samdutton :-)
(also friendly ping)

Reply:
Feel free to remove.